hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -62,33 +62,32 @@
   private long lastFlushOffset;
   private long lastTotalAppendOffset = 0;
 
-  private final int bufferSize;
+  private static int bufferSize;
```

Comment:
Why should this be static and not final ?

Reply:
static final variables cannot be initialised in constructor. And we cannot initialise the same either during declaration or in a static block.